src/base-command.ts

```diff
@@ -15,13 +15,9 @@ export default abstract class extends Command {
     help: flags.help({ char: 'h' }),
   };
 
-  private getFormattedJSON(obj: object) {
-    return JSON.stringify(obj, null, 2);
-  }
-
   async init() {
     if (!this.app) {
-      this.app = await AppService.create(this.config.configDir);
+      this.app = await AppService.create(this.config.configDir, this.config.userAgent.split(/\/|\s/g)[2]);
```

Comment:
Nice 👍 , presumably this will grab this: `0.4.10-rc.1`?

Worth sending the entire userAgent in another header (eg. `@architect-io/cli/0.4.10-rc.1 darwin-x64 node-v13.3.0`) for analytics purposes? scope creep...

Reply:
I had done that originally, but didn't really think the extra information was that useful. When we do, maybe we can send it to Segment or something, but that data currently has no place to go anyway

Reply:
Fair enough... this is me just being greedy. Agreed, probably not worth it for now.

Will be useful someday when we encounter issues that we can't reproduce and want to know environment details, but we're probably a long ways away from that right now since we can just ask the friendly what their userAgent is.